src/Jackalope/Transport/DoctrineDBAL/Client.php

```diff
@@ -2201,6 +2202,11 @@ public function getBinaryStream($path)
             array($nodeId, $propertyName, $this->workspaceName)
         );
 
+        if (count($data) === 0) {
+            throw new RepositoryException('No binary data found in stream');
```

Comment:
Mostly sanity check.
If you run the test w/o this commit it will fail with an error
>fstat() expects parameter 1 to be resource, boolean given

because the node itself has been copied, but its binary data wasn't and BinaryStreamWrapper will get this error on init.